modules/net-vpn-ha/variables.tf

```diff
@@ -63,8 +64,9 @@ variable "router_config" {
       all_subnets = bool
       ip_ranges   = map(string)
     }))
-    keepalive = optional(number)
-    name      = optional(string)
+    keepalive     = optional(number)
+    name          = optional(string)
+    override_name = optional(string, null)
```

Comment:
still using override name in this file

Reply:
Hey mate, sorry I don't understand what you want me to do with this, it's been a few weeks since we've worked on this. Can you give me a quick heads up and I'll get this fixed please?